learning-bitmark/working-with-bitmarks/issuing-bitmarks.md

```diff
@@ -0,0 +1,183 @@
+        <img src="images/RegisteringProperties_0.png" alt="Properties screen" title="Properties screen" width="250" style="padding: 20px" />
+        <img src="images/RegisteringProperties_1.png" alt="Properties Register screen" title="Properties Register screen" width="250" style="padding: 20px" />
+    </div>
+
+    <br>
+
+* Tap **PHOTOS** or **FILES** to browse the desired asset
+
+    > It requires permission granted to access `Photos` and/or `Files`
+
+    <div style="background-color: #efefef; text-align: center;">
+        <img src="images/RegisteringProperties_2.png" alt="Grant Permission popup" title="Grant Permission popup" width="250" style="padding: 20px" />
+        <img src="images/RegisteringProperties_3.png" alt="Open Photos action sheet" title="Open Photos action sheet" width="250" style="padding: 20px" />
+        <img src="images/RegisteringProperties_4.png" alt="PHOTOS browsed" title="PHOTOS browsed" width="250" style="padding: 20px" />
+    </div>
+
+    <br>
+
+* Fill in the required information
+
+    > As soon as the desired asset is selected, the app computes the asset's fingerprint and opens REGISTER PROPERTY RIGHTS screen which allows users to provide more detailed information for the asset.
+
+    >* Currently, the `PROPERTY NAME` and `NUMBER OF BITMARKS TO ISSUE` fields are mandatory.
+
+    >* For each `Asset`, we are able to issue multiple `Bitmark Certificates` - defined by  `number of bitmarks` in the REGISTER PROPERTY RIGHTS screen.
+
+    <div style="background-color: #efefef; text-align: center;">
+        <img src="images/RegisteringProperties_5.png" alt="Register Property Rights screen" title="Register Property Rights screen" width="250" style="padding: 20px" />
+        <img src="images/RegisteringProperties_6.png" alt="Fill in required information" title="Fill in required information" width="250" style="padding: 20px" />
+    </div>
+
+    <br>
+
+* Tap **ISSUE** button
+
+    >As soon as users tap **ISSUE**, the app submits both the `Asset Registration` request and `bitmarks Issuance` transactions to the Bitmark network. It would takes few seconds for the submission to be successful. After that the properties will be added to the PROPERTIES > YOURS screen.
+    > 
+    > **NOTE:** It will takes several minutes for the transactions to be confirmed on the Bitmark blockchain after submitted. 
+
+<div style="background-color: #efefef; text-align: center;">
+    <img src="images/RegisteringProperties_7.png" alt="Submitting transaction" title="Submitting transactions" width="250" style="padding: 20px" />
+    <img src="images/RegisteringProperties_8.png" alt="Submission succeeded" title="Submission succeeded" width="250" style="padding: 20px" />
+    <img src="images/RegisteringProperties_9.png" alt="Properties added" title="Properties added" width="250" style="padding: 20px" />
+</div>
+
+<br>
+
+* [Verify](https://github.com/bitmark-inc/docs/) bitmark transactions.
```

Comment:
I assume that's going to be a link to a future document?

Reply:
Yes.